### ISSUE TITLE
@requires is not printed

### ISSUE THREAD

**Issue by AndyOGo** [jsdoc2md/jsdoc-to-markdown#70]
The [@requires](http://usejsdoc.org/tags-requires.html) tag is not printed if defined.

I'm using [gulp-jsdoc-to-markdown](https://www.npmjs.com/package/gulp-jsdoc-to-markdown) version 1.2.1 which uses [jsdoc-to-markdown](https://www.npmjs.com/package/jsdoc-to-markdown) version 1.3.3.

Works as expected with [gulp-jsdoc3](https://www.npmjs.com/package/gulp-jsdoc3)

Source:
````javascript
define('foo', ['required'], function(required) {
    /**
     * Module foo
     * @exports foo
     */
    var foo = {
        /**
         * The module's `color` property.
         *
         * @requires module:required
         * */
        bar: function() {
            // using required here
        }
    };

    return foo;
});
```

Output:
````markdown
<a name="module_foo"></a>
## foo
Module foo

<a name="module_foo.bar"></a>
### foo.bar()
The module's `color` property.

**Kind**: static method of <code>[foo](#module_foo)</code>
````

**Comment by 75lb** [comment]
Thanks, will fix tonight

**Comment by AndyOGo** [comment]
Thank you for your quick reply and your awesome tool:)
I'm keen to use the new version.

**Comment by 75lb** [comment]
fixed and released in dmd v1.3.8.. update your jsdoc2md tool/plugin to pick it up.. thanks again.. i'll let you know once jsdoc2md version 2 is in beta.

**Comment by AndyOGo** [comment]
Awesome, thank you very much for your quick release. Works great:)
Cool, yeah, just through me a message.
